test/extern/diten/externfaston.chpl

```diff
@@ -0,0 +1,27 @@
+use CommDiagnostics;
+require "externfaston.c", "externfaston.h";
+
+pragma "fast-on safe extern function"
+extern proc foo1(): int;
+extern proc foo2(): int;
+
+pragma "fast-on safe extern function"
+proc nonExternFunction() {
```

Comment:
Whoops, I know I said I was done reviewing, but I just noticed this: Did you intend to use nonExternFunction somewhere to show that we ignore that fast on safe for Chapel functions?